Safety/out-of-hours-support.rst

```diff
@@ -0,0 +1,88 @@
+.. This is a template for operational procedures. Each procedure will have its own sub-directory. This comment may be deleted when the template is copied to the destination.
+
+.. Review the README in this procedure's directory on instructions to contribute.
+.. Static objects, such as figures, should be stored in the _static directory. Review the _static/README in this procedure's directory on instructions to contribute.
+.. Do not remove the comments that describe each section. They are included to provide guidance to contributors.
+.. Do not remove other content provided in the templates, such as a section. Instead, comment out the content and include comments to explain the situation. For example:
+	- If a section within the template is not needed, comment out the section title and label reference. Include a comment explaining why this is not required.
+    - If a file cannot include a title (surrounded by ampersands (#)), comment out the title from the template and include a comment explaining why this is implemented (in addition to applying the ``title`` directive).
+
+.. Include one Primary Author and list of Contributors (comma separated) between the asterisks (*):
+.. |author| replace:: *Alysha Shugart*
+
+.. This is the label that can be used as for cross referencing this procedure.
+.. Recommended format is "Directory Name"-"Title Name"  -- Spaces should be replaced by hyphens.
+.. _Safety-out-of-hours-support:
+.. Each section should includes a label for cross referencing to a given area.
+.. Recommended format for all labels is "Title Name"-"Section Name" -- Spaces should be replaced by hyphens.
+.. To reference a label that isn't associated with an reST object such as a title or figure, you must include the link an explicit title using the syntax :ref:`link text <label-name>`.
+.. An error will alert you of identical labels during the build process.
+.. _`Emergency Call List`: https://start.1password.com/open/i?a=EJGC5QLBFNAJ3ODCNVC2I3HIPM&v=mqlr5qdkpipoyamzhty5kg365y&i=tu4v5aa4tvhbm4e3rc5siiwada&h=lsstit.1password.com
+.. _`download`: https://1password.com/downloads
+.. _`IT support`: https://jira.lsstcorp.org/projects/IHS/issues/IHS-6753?filter=allopenissues
+.. _`#auxtel-operations`: https://lsstc.slack.com/archives/CLQ63T89L
+.. _`#auxtel-virtual-control-room`: https://lsstc.slack.com/archives/C01K4M6R4AH
+.. _`#rubin-observing`: https://lsstc.slack.com/archives/C04HULH5HHD
+
+####################
+Out of hours support
+####################
+
+The Out of Hours support call list is managed in a 1Password vault named "Emergency Call List".
+All observing specialists are automatically enrolled in the vault once they have a 1Password account.
```

Comment:
this isn't quite true...  Anastasia or myself need to add them manually. So maybe modify the text so as they get an 1password account from IT, then request access from myself or Anastasia.

Reply:
👍